tempy/widgets.py

```diff
@@ -181,11 +182,78 @@ def make_footer(self, footer):
         self._make_table_part('footer', footer)
 
     def make_caption(self, caption):
-        """Adds/Sobstitute the table's caption."""
+        """Adds/Substitutes the table's caption."""
         if not hasattr(self, 'caption'):
             self(caption=Caption())
         return self.caption.empty()(caption)
 
+    def col_class(self, css_class, col_index=None):
+        # adds css_class to every cell
+        if col_index is None:
+            for row in self.body.childs:
+                for cell in row.childs:
+                    cell.attr(klass=css_class)
+        elif col_index >= 0:
+            for row in self.body.childs:
+                if col_index < len(row.childs):
+                    row.childs[col_index].attr(klass=css_class)
+
+    def row_class(self, css_class, row_index=None):
+        # adds css_class to every row
+        if row_index is None:
+            for row in self.body.childs:
+                row.attr(klass=css_class)
+        elif row_index >= 0 and (row_index < len(self.body.childs)):
+            self.body.childs[row_index].attr(klass=css_class)
```

Comment:
Same raise here and in the methods below.. maybe a _check_valid_index decorator or method can be implemented..